### DISCUSSION TITLE
Deleting articles

### DISCUSSION THREAD

**Discussion by arvigeus** [Roznoshchik/Lurnby#16] [Ideas]
I could not find a button to do it. The closest thing I found was "Archive" where the article disappears into undisclosed location.

Offtopic: Great app! I am thinking of replacing Pocket with it.

**Comment by Roznoshchik** [comment]
Hey yeah. Deleting articles is a `soft delete` for now. It removes them from your list but doesn't delete them entirely. 

Part of the reason is that I wanted to introduce an `archived` section at some point where you could `undo` something or `fully delete` it. 

But I haven't gotten around to that yet. 

The second reason is that I need to think through the logic of highlights from deleted articles. The highlights would likely stay even if the article was deleted, but they would be a bit broken.

I think once I decouple highlights from articles to make them more independent, then deleting articles will run a script on all article highlights first and ensure that nothing gets broken. 

Is there a reason that the "soft delete" isn't enough for you atm?

just fyi, deleting your account fully deletes everything, not soft deletes.

**Comment by arvigeus** [comment]
> Is there a reason that the "soft delete" isn't enough for you atm?

My use case was i was unable to parse an article, so I created a new manual entry, then tried to remove the old one.

Thank you for the clarification! Since you are working on this, you may close this issue any time you want.
